xarray/tests/test_backends_datatree.py

```diff
@@ -455,32 +466,29 @@ def test_to_zarr_not_consolidated(self, tmpdir, simple_datatree):
                 assert_equal(original_dt, roundtrip_dt)
 
     def test_to_zarr_default_write_mode(self, tmpdir, simple_datatree):
-        import zarr
-
-        simple_datatree.to_zarr(tmpdir)
+        simple_datatree.to_zarr(str(tmpdir))
 
         # with default settings, to_zarr should not overwrite an existing dir
-        with pytest.raises(zarr.errors.ContainsGroupError):
-            simple_datatree.to_zarr(tmpdir)
+        with pytest.raises(FileExistsError):
```

Comment:
I feel like this error type shouldn't need to be changed, but again I think it's zarr-python who's behavior changed, see https://github.com/zarr-developers/zarr-python/issues/2821#issuecomment-2702287158